.env

```diff
@@ -10,3 +10,6 @@ REACT_PUBLIC_URL=https://care.ohc.network
 ESLINT_NO_DEV_ERRORS=true
 
 CARE_CDN_URL="https://egov-s3-facility-10bedicu.s3.amazonaws.com https://egov-s3-patient-data-10bedicu.s3.amazonaws.com http://localhost:4566"
+
+#.env
+CARE_API=http://127.0.0.1:9000
```

Comment:
Could you discard this change? 

You can use `.env.local` file to set envs for local development as that file is git ignored.

Reply:
Done !!